app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user, @comment.post)
+    else
+      render :create
+    end
+  end
+
```

Comment:
- [ ] Kindly add a new method in your comments controller as done in the posts controller, it's a project requirement. Also, ensure it has a corresponding router in the post controller routes.

Reply:
Hey but the forms are just for the posts and comments 
To view the forms the route is http://127.0.0.1:3000/users/1/posts/new